### PR TITLE
Added erc-1155 article

### PR DESCRIPTION
**Pull request by tanim0la**

### CODE REVIEW

File: erc-1155-multi-token-standard/ERC-1155 Multi Token Standard 7598a340ece84ec89c0529692f085c27.md [hunk 1]
```diff
@@ -0,0 +1,947 @@
...
+
+Using the above code, Bob could call the `balanceOf` function with the token ID `(2 << 128) + 7` to check his ownership:
+
+```solidity
+uint256 rareSkillsTokenCollectionID = 2 << 128; // collection id is 2
+uint256 rsNFT = 7; // item id
+
+// Returns 1 if Bob owns the tokenid passed, else, 0
+uint256 bobBalance = balanceOf(
+											address(Bob),
```
Comment (RareSkills) [inline erc-1155-multi-token-standard/ERC-1155 Multi Token Standard 7598a340ece84ec89c0529692f085c27.md]: something seems off with the formatting here

File: erc-1155-multi-token-standard/ERC-1155 Multi Token Standard 7598a340ece84ec89c0529692f085c27.md [hunk 2]
```diff
@@ -0,0 +1,947 @@
...
+
+Below is an example code on how to compute:
+
+1. the ERC-1155 token ID given the NFT collection ID and itemId
+2. the collection ID and item ID, given the ERC-1555 token ID
+
+```solidity
+contract A {
+
+		// 1. COMPUTE TOKEN ID
```
Comment (RareSkills) [inline erc-1155-multi-token-standard/ERC-1155 Multi Token Standard 7598a340ece84ec89c0529692f085c27.md]: again, formatting is off. Maybe mixed tabs?

File: erc-1155-multi-token-standard/ERC-1155 Multi Token Standard 7598a340ece84ec89c0529692f085c27.md [hunk 3]
```diff
@@ -0,0 +1,947 @@
...
+        uint256 itemId 
+        ) public pure returns (bytes32 tokenId) {
+        // shift the collection id by 128 to the left
+        uint256 shiftedCollectionId = collectionId << 128;
+
+        // add the item id to the shifted collection id
+        tokenId =  bytes32(shiftedCollectionId + itemId);
+    }
+
+		// 2. GET COLLECTION ID AND ITEM ID
```
Comment (RareSkills) [inline erc-1155-multi-token-standard/ERC-1155 Multi Token Standard 7598a340ece84ec89c0529692f085c27.md]: same

File: erc-1155-multi-token-standard/ERC-1155 Multi Token Standard 7598a340ece84ec89c0529692f085c27.md [hunk 4]
```diff
@@ -0,0 +1,947 @@
...
+    */
+    function safeBatchTransferFrom(
+    			address _from,
+    			address _to,
+    			uint256[] calldata _ids,
+    			uint256[] calldata _values,
+    			bytes calldata _data) external;
+    ```
+    
+    Furthermore, the standard allows for owners and operators to execute batch transfers. Multiple sets of tokens can be transferred from a source address to a destination address in a single transaction.
```
Comment (RareSkills) [inline erc-1155-multi-token-standard/ERC-1155 Multi Token Standard 7598a340ece84ec89c0529692f085c27.md:281]: Not sure this should be indented

Comment (tanim0la) [reply erc-1155-multi-token-standard/ERC-1155 Multi Token Standard 7598a340ece84ec89c0529692f085c27.md:281]: it should be

**Review by RareSkills — APPROVED** [review]

**Pull request merged**